### PR TITLE
Documentation Rework in Master Page

### PR DESCRIPTION
**Pull request by prablr79**

### CODE REVIEW

File: docs/manual_install_guide_hpe_3par_plugin_with_openshift_kubernetes.md [hunk 1]
```diff
@@ -24,24 +24,18 @@ This document details the installation steps in order to get up and running quic
...
-| Kubernetes 1.10.3                                       | Yes                              | 17.03                 | 3.2.2 MU6+ P107 & 3.3.1 MU1, MU2 |
-| OpenShift 3.7 RPM based installation (Kubernetes 1.7.6) | Yes                              | 1.12.6                | 3.2.2 MU6+ P107 & 3.3.1 MU1, MU2   |
+* [Support Matrix for Kubernetes and Openshift](https://spock.corp.int.hpe.com/spock/utility/document.aspx?docurl=Shared%20Documents/hw/3par/3par_volume_plugin_for_docker.pdf)
 
 **NOTE**
-  * Managed Plugin is not supported for Kubernetes or Openshift 3.7
+  * Managed Plugin is not supported for Kubernetes or Openshift 
 
-  * The install of OpenShift for this paper was done on RHEL 7.4. Other versions of Linux may not be supported.
+  * The install of OpenShift for this paper was done on RHEL 7.x. Other versions of Linux may not be supported.
```
Comment (nilangekarss) [inline docs/manual_install_guide_hpe_3par_plugin_with_openshift_kubernetes.md]: Shall we mention 7.x or all supported and qualified versions?

File: docs/troubleshooting.md [hunk 2]
```diff
@@ -32,6 +32,11 @@ For setting up secured etcd cluster, refer this doc:
 
 Sometimes it is useful to get more verbose output from the plugin. In order to do this one must change the logging property to be one of the following values: INFO, WARN, ERROR, DEBUG.
 
+REST Calls made to 3PAR Array from Volume Plugin enable
```
Comment (nilangekarss) [inline docs/troubleshooting.md]: It is better to mention  "To enable logging for REST calls made to 3PAR array from Volume Plugin use below flag"

File: docs/troubleshooting.md [hunk 3]
```diff
@@ -50,3 +55,43 @@ If no volumes are in mounted state and `lsscsi` lists any 3PAR data volumes then
...
+for any older version below v3.1
+
+```
+/var/log/messages
+```
+#### Containerized Plugin 
+
+```
+$docker logs -f <container id of Plugin> 
+container id of plugin can got by $docker ps -a | grep hpe
```
Comment (nilangekarss) [inline docs/troubleshooting.md]: Getting container id of plugin:  docker ps -a | grep hpe

File: docs/troubleshooting.md [hunk 4]
```diff
@@ -50,3 +55,43 @@ If no volumes are in mounted state and `lsscsi` lists any 3PAR data volumes then
...
+#### Containerized Plugin 
+
+```
+$docker logs -f <container id of Plugin> 
+container id of plugin can got by $docker ps -a | grep hpe 
+```
+
+ ## Capturing Logs in Kubernetes/OpenShift environments
+ 
+ Collect above Containerized Plugin logs along with than following logs are required.
```
Comment (nilangekarss) [inline docs/troubleshooting.md]: Collect above Containerized Plugin logs along with the following logs.

File: docs/troubleshooting.md [hunk 5]
```diff
@@ -50,3 +55,43 @@ If no volumes are in mounted state and `lsscsi` lists any 3PAR data volumes then
...
+ 
+ ```
+ /var/log/dory.log
+ ```
+ 
+ Note: From all the nodes in the Kubernetes/OpenShift Cluster.
+ 
+ ### Dynamic Provisioner Hang 
+ 
+ if you observ any doryd hang in your system, following command need to run to bring back online.
```
Comment (nilangekarss) [inline docs/troubleshooting.md]: observe spell correction

File: docs/manual_install_guide_hpe_3par_plugin_with_openshift_kubernetes.md (hunk 1 above)
Comment (prablr79) [reply docs/manual_install_guide_hpe_3par_plugin_with_openshift_kubernetes.md]: its better to call out 7.x., we do support most of the 7.* series of RHEL.

File: docs/troubleshooting.md (hunk 2 above)
Comment (prablr79) [reply docs/troubleshooting.md]: updated

File: docs/troubleshooting.md (hunk 3 above)
Comment (prablr79) [reply docs/troubleshooting.md]: updated

File: docs/troubleshooting.md (hunk 4 above)
Comment (prablr79) [reply docs/troubleshooting.md]: done updated typo

File: docs/troubleshooting.md (hunk 5 above)
Comment (prablr79) [reply docs/troubleshooting.md]: updated! Thanks

**Comment by nilangekarss** [comment]
Merging these changes. These will be updated on need basis.

**Pull request merged**